pkg/centos7/python_patch_3.9.2.patch

```diff
@@ -0,0 +1,98 @@
+diff -aur a/Lib/ssl.py Lib/ssl.py
+--- a/Lib/ssl.py	2020-10-05 15:07:58.000000000 +0000
++++ Lib/ssl.py	2021-03-02 04:23:32.026226000 +0000
```

Comment:
Is this correct? Looks like you have copied the git diff here which might result in syntax errors.

Reply:
Yes its working